godotenv.go

```diff
@@ -37,10 +37,8 @@ func Load(filenames ...string) (err error) {
 
 	for _, filename := range filenames {
 		err = loadFile(filename)
-		if err != nil {
-			return // return early on a spazout
-		}
 	}
+	// return the last load error
 	return
```

Comment:
Yeah, I guess I'm changing the meaning of Load to be "load any of these which exist, in precedence order".  Which is a useful function to have...in my project's tests, I load 3 files like this (.env.local, .env.test, and .env).

I'd suggest either 

1. keeping the new definition, and letting the user call Load() for each file if they care about errors
2. do 1. and create a new "MustLoad" function using the old semantics
3.  revert Load and create a "LoadAny" function with the new semantics.  But if we do this, existing code will break.  They'll call Load() with no args, which will try and load ".env.local" and blow up.  So I'd probably revert filenamesOrDefault() as well, and add a new anyFilenamesOrDefault() function which returns the ".env.local".

Also, I should have done the same thing to Read().